Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/PowerTransmissions/GearBox.cs

```diff
@@ -248,19 +253,41 @@ public void AutoAtGear()
             gearedDown = false;
         }
 
+        /// <summary>
+        /// ClutchOn is true when clutch is fully engaged, and false when slipping
+        /// </summary>
         public bool clutchOn;
         public bool IsClutchOn
         {
             get
             {
-                if (DieselEngine.Locomotive.ThrottlePercent > 0)
+                if (Gears[0].TypeGearBox == 1)
```

Comment:
In some cases, the variable is called TypeGearBox, in others, it is called GearBoxType.

This should be renamed GearBoxType